### PR TITLE
releng - 0.9.16.0 version increment, deps rebase

### PR DESCRIPTION
**Pull request by thisisshi**

### CODE REVIEW

File: tests/test_policy.py [hunk 1]
```diff
@@ -315,50 +317,56 @@ def test_config_resource_support(self):
         # of a resource.
 
         whitelist = {
-            'AWS::Kinesis::StreamConsumer',
-            'AWS::CodeDeploy::DeploymentConfig',
-            'AWS::OpenSearch::Domain',  # this is effectively an alias
+            'AWS::ApiGatewayV2::Api',
```
Comment (thisisshi) [inline tests/test_policy.py:320]: this is a sorted list of the left + some new entries (Launch Template, EMR, SageMaker, Guardduty entires)

File: tools/c7n_mailer/tests/common.py [hunk 2]
```diff
@@ -44,7 +44,7 @@
     'role': 'arn:aws:iam::xxxx:role/cloudcustodian-mailer',
     'ldap_uid_tags': ['CreatorName', 'Owner'],
     'templates_folders': [os.path.abspath(os.path.dirname(__file__)),
-                          os.path.abspath('/')],
+                          os.path.abspath('/'), ''],
```
Comment (thisisshi) [inline tools/c7n_mailer/tests/common.py:47]: the posixpath.join and os.path.join behavior differential causes us to do a look up on files like `C:\\C:/foo/bar/example.jinja`. We pass a empty path here to ensure that we are able to do an abs path join

**Review by kapilt — APPROVED** [review]
thanks, lgtm

**Review by ajkerrigan — APPROVED** [review]
Thanks for wrestling through this one :+1:

**Pull request merged**